ecs-agent/api/attachment/resource/ebs_discovery_windows.go

```diff
@@ -42,18 +42,18 @@ func (api *EBSDiscoveryClient) ConfirmEBSVolumeIsAttached(deviceName, volumeID s
 		return "", errors.Wrapf(err, "failed to run ebsnvme-id.exe: %s", string(output))
 	}
 
-	_, err = parseExecutableOutput(output, volumeID, deviceName)
+	actualDeviceName, err := parseExecutableOutput(output, volumeID, deviceName)
 	if err != nil {
 		return "", errors.Wrapf(err, "failed to parse ebsnvme-id.exe output for volumeID: %s and deviceName: %s",
 			volumeID, deviceName)
 	}
 
-	log.Info(fmt.Sprintf("found volume with volumeID: %s and deviceName: %s", volumeID, deviceName))
+	log.Info(fmt.Sprintf("found volume with volumeID: %s and deviceName: %s", volumeID, actualDeviceName))
 
-	return "", nil
+	return actualDeviceName, nil
 }
 
-// parseExecutableOutput parses the output of `ebsnvme-id.exe` and returns the volumeId.
+// parseExecutableOutput parses the output of `ebsnvme-id.exe` and returns the deviceName.
 func parseExecutableOutput(output []byte, candidateVolumeId string, candidateDeviceName string) (string, error) {
```

Comment:
nit: We should also remove `candidateDeviceName` here since it's no longer being used.